R/plot-cumulative-time-profile.R

```diff
@@ -99,15 +99,25 @@ plotCumulativeTimeProfile <- function(data = NULL,
 
   # And optional color palette otherwise use colors from theme
   if (!isEmpty(plotConfiguration$colorPalette)) {
-    try(suppressMessages(
-      plotObject <- plotObject +
-        ggplot2::scale_fill_brewer(
-          palette = plotConfiguration$colorPalette,
-          aesthetics = c("color", "fill")
-        )
-    ))
+    if(!isIncluded(plotConfiguration$colorPalette, .ViridisPalettes)){
+      try(suppressMessages(
+        plotObject <- plotObject +
+          ggplot2::scale_fill_brewer(
+            palette = plotConfiguration$colorPalette,
+            aesthetics = c("color", "fill")
+          )
+      ))
+    }
+    if(isIncluded(plotConfiguration$colorPalette, .ViridisPalettes)){
```

Comment:
why not just `else `instead of `if(isIncluded`

Reply:
also I would exchange the If and the else part (just to avoid the negation in the if):
```
if(isIncluded(plotConfiguration$colorPalette, .ViridisPalettes)){
...
else {
...
}
```